### ISSUE TITLE
Wave removes startupProbe

### ISSUE THREAD

**Issue by moonpie79** [wave-k8s/wave#118]
startupProbes went GA in kubernetes 1.20, but even with the latest version of Wave, when I try to create a deployment with one added, it is removed by Wave, presumably because it's complied against an older version of the Kubernetes API.

e.g. 

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: my-chart
  labels:
    helm.sh/chart: my-chart-0.1.0
    app.kubernetes.io/name: my-chart
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
  annotations:
    wave.pusher.com/update-on-config-change: "true"
spec:
  replicas: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: my-chart
  template:
    metadata:
      labels:
        app.kubernetes.io/name: my-chart
    spec:
      securityContext:
        {}
      containers:
        - name: my-chart
          securityContext:
            {}
          image: "nginx:1.16.0"
          imagePullPolicy: IfNotPresent
          ports:
            - name: http
              containerPort: 80
              protocol: TCP
          livenessProbe:
            httpGet:
              path: /
              port: http
          readinessProbe:
            httpGet:
              path: /
              port: http
          startupProbe:
            httpGet:
              path: /
              port: http
          resources:
            {}
```

when applied to the cluster doesn't have the startupProbe

**Comment by jan-kantert** [comment]
Will be fixed by #145

**Comment by toelke** [comment]
Fixed with #145
